.github/workflows/test.yml

```diff
@@ -12,23 +13,24 @@ jobs:
     timeout-minutes: 100
     runs-on: ubuntu-latest
     env:
-      PROJECT_NAME: 'libbpf'
       AUTHOR_EMAIL: "$(git log -1 --pretty=\"%aE\")"
       KERNEL: LATEST
-
+      REPO_ROOT: ${{ github.workspace }}
+      REPO_PATH: ""
     steps:
       - uses: actions/checkout@v2
-      - name: install
-        run: sudo apt-get update && sudo apt-get install aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils
-      - name: Get bpf-next source
-        if: ${{ github.repository == 'kernel-patches/vmtest' }}
-        uses: actions/checkout@v2
+      - if: ${{ github.repository != 'kernel-patches/bpf' }}
+        name: Download bpf-next tree
+        uses: libbpf/ci/get-linux-source@master
         with:
-          repository: kernel-patches/bpf
-          ref: 'bpf-next'
-          path: 'linux'
-      - name: Move linux source in place
-        if: ${{ github.repository == 'kernel-patches/vmtest' }}
-        run: rsync --exclude .git -av linux/. . && rm -rf linux
-      - name: Kernel LATEST + selftests
-        run: export REPO_ROOT=$GITHUB_WORKSPACE; export CI_ROOT=$REPO_ROOT/travis-ci; export VMTEST_ROOT=$CI_ROOT/vmtest; ${CI_ROOT}/vmtest/run_vmtest.sh || exit 1
+          dest: '.kernel'
```

Comment:
why not download right into the destination? did you run into any problems downloading sources to the final destination directory?

Reply:
Yeah, the script doesn't do well direct extracting to root directory, so for now i just left this as is